src/games/strategy/triplea/delegate/AbstractUndoableMove.java

```diff
@@ -26,11 +27,32 @@
   protected int m_index;
   protected final Collection<Unit> m_units;
 
+  public AbstractUndoableMove(final Collection<Unit> units) {
+    this(new CompositeChange(), units);
+  }
+
+
   public AbstractUndoableMove(final CompositeChange change, final Collection<Unit> units) {
     m_change = change;
     m_units = units;
   }
 
+  public boolean containsAnyUnit(Set<Unit> units) {
```

Comment:
This method name just seems a little off to me. Maybe it should just be 'containsAnyOf' or a longer option would be 'containsAnyOfTheseUnits'